src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -213,6 +214,8 @@
 		list-style: none;
 		padding: 0;
 		margin: 0;
+		max-height: calc(~"100vh - "2 * @dropdown-list-margin);
```

Comment:
I think that better use simple `2 * 6 * @px_base` instead of additional variables.

Reply:
Wouldn't be better to have variable for this? So we will have one place to change margin and after this container position and height are updated as well?

Reply:
I understand you point of view but you need search this value across the project now. In my opinion, if you want to leave this variable you should define it in this file eg. on beginning.

Reply:
Ok, moved to the beginning of this file.